Loenn/entities/strawberry_jam/flag_breaker_box.lua

```diff
@@ -0,0 +1,37 @@
+local utils = require("utils")
+local drawableSprite = require("structs.drawable_sprite")
+
+local breakerBox = {}
+
+breakerBox.name = "CommunalHelper/SJ/FlagBreakerBlock"
+breakerBox.depth = -10550
+breakerBox.texture = "objects/breakerBox/Idle00"
+breakerBox.fieldInformation = {
+    music_progress = {
+        fieldType = "integer",
+    }
+}
+breakerBox.placements = {
+    name = "breaker_box",
+    data = {
+        flipX = false,
+        music_progress = -1,
+        music_session = false,
+        music = "",
+        flag = false
```

Comment:
Should flag be a string?
And should we add aliveState?
```suggestion
        flag = "",
        aliveState = true
```